### PR TITLE
Publish Kubernetes CVEs: CVE-2020-{8554,8567,8568,8569,8570}

### PR DESCRIPTION
**Pull request by tallclair**

### CODE REVIEW

**Review by ritazh — APPROVED** [review]
CVE-2020-8567 and CVE-2020-8568 LGTM

File: 2020/8xxx/CVE-2020-8568.json [hunk 1]
```diff
@@ -1,18 +1,113 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "security@kubernetes.io",
+        "DATE_PUBLIC": "2020-11-16T21:00:00.000Z",
```
Comment (aramase) [inline 2020/8xxx/CVE-2020-8568.json:4]: Date public for this CVE is `2020-11-10T21:00:00.000Z`

**Review by joelsmith — CHANGES_REQUESTED** [review]
Per https://github.com/kubernetes/security/blob/master/cna-handbook.md#populate-cve-details we should begin the title and descriptions with Kubernetes <component>. I have provided suggested rewording which you can use or ignore, but I think that it's important to mention Kubernetes so that it will be clear what is affected in global CVE lists.

**Comment by tallclair** [comment]
Thanks Joel! All feedback integrated. I'll merge & open the upstream PR

**Pull request merged**